libbeat/outputs/mode/balance.go

```diff
@@ -231,6 +233,7 @@ func (m *LoadBalancerMode) onFail(msg eventsMessage, err error) {
 	logp.Info("Error publishing events (retrying): %s", err)
 
 	if ok := m.forwardEvent(m.retries, msg); !ok {
+		messagesDropped.Add(1)
```

Comment:
maybe add a method 'dropping(msg eventsMessage)' to LoadBalancerMode, so we have less repetitive code and add more metrics at this place later if required.